x-pack/plugin/wildcard/src/yamlRestTest/resources/rest-api-spec/test/10_script_values.yml

```diff
@@ -0,0 +1,42 @@
+setup:
+  - do:
+      indices.create:
+        index:  test
```

Comment:
set the number of shards to 1 to avoid doc routing issues.

Reply:
Done.